doc/quickstart_in_rviz/quickstart_in_rviz_tutorial.rst

```diff
@@ -134,8 +134,10 @@ Moving Joints or in Null Space
 ++++++++++++++++++++++++++++++
 You can use the **Joints** tab to move single joints and the redundant joints of 7-DOF robots. Try moving the "null space exploration" slider as shown in the animation below.
 
-.. image:: rviz_joints_nullspace.webm
+.. video:: ../../_static/rviz_joints_nullspace.webm
```

Comment:
to @AndyZe 's point, youtube has just been embedded with HTML. wouldn't that be a lot easier than a custom extension?

```
.. raw:: html

    <object width="700" height="400">
      <embed
        src="https://www.youtube.com/v/8sOucNloJeI&hl=en_US&fs=1&rel=0"
        type="application/x-shockwave-flash" allowscriptaccess="always"
        allowfullscreen="true" width="700"
        height="385">
      </embed>
    </object>
```